lib/Doctrine/DBAL/Types/JsonArrayType.php

```diff
@@ -70,12 +51,5 @@ public function getName()
     {
         return Type::JSON_ARRAY;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function requiresSQLCommentHint(AbstractPlatform $platform)
```

Comment:
But these are both json structures, so basically what should I use instead for this deprecation ?